sql/core/src/main/scala/org/apache/spark/sql/jdbc/README.md

```diff
@@ -46,6 +46,12 @@ so they can be turned off and can be replaced with custom implementation. All CP
 which must be unique. One can set the following configuration entry in `SparkConf` to turn off CPs:
 `spark.sql.sources.disabledJdbcConnProviderList=name1,name2`.
 
+## How to enforce a specific JDBC connection provider?
+
+When more than one JDBC connection provider can handle a specific driver and options, it is possible to
+disambiguate and enforce a particular CP for the JDBC data source. One can set the DataFrameReader
```

Comment:
Is this related to the reader only? how about the writer?

Reply:
+1 on this question.

Reply:
It is both. I meant a DataFrame option here.